src/page/dataverse/component/generalMetadata/generalMetadata.tsx

```diff
@@ -0,0 +1,57 @@
+import type { FC } from 'react'
+import { memo } from 'react'
+import { useTranslation } from 'react-i18next'
+import type { IconName } from '@/component/icon/icon'
+import { Icon } from '@/component/icon/icon'
+import './generalMetadata.scss'
+import './i18n/index'
+
+export type GeneralMetadataTitle =
+  | 'format'
+  | 'topic'
+  | 'license'
+  | 'geographicalCoverage'
+  | 'temporalCoverage'
+  | 'category'
+
+export type GeneralMetadata = {
+  iconName: IconName
+  title: GeneralMetadataTitle
```

Comment:
We cannot specify the different titles in the component because the title value depends on the caller. For example it makes no sense for the data space page to have the format title because a data space does not have a format meta data